templates/email/artist_apply.email

```diff
@@ -28,31 +37,40 @@
     {% if instagram %}
         Instagram: {{ instagram }}
     {% endif %}
-    Link to music: {{ music_link }}
+    SoundCloud: {{ soundcloud }}
```

Comment:
It looks like `SoundCloud` is duplicated here now

Reply:
Ahh. Yes, you are right. Just pushed though a fix